consoles/sshVirtsh.pm

```diff
@@ -496,6 +499,33 @@ sub resume ($self) {
 
 sub get_remote_vmm ($self) { $bmwqemu::vars{VMWARE_REMOTE_VMM} // '' }
 
+sub _encode_config ($self, $file) {
+    unless (!!$file) {
+        croak "Please provide a configuration file to encode!";
+    }
+
+    my $conf = sprintf('%s/data/%s', $bmwqemu::vars{CASEDIR}, $file);
+    my $encoded_config;
+
+    {
+        open(my $fh_conf, "<", $conf);
+
+        my $z = IO::Compress::Gzip->new('test', Level => Z_BEST_COMPRESSION);
+        $z->print(<$fh_conf>);
+        $z->close();
+        close $fh_conf;
+
+        open(my $gzip, '<', 'test');
+        local $/ = undef;
+        my $data = <$gzip>;
+        close $gzip;
```

Comment:
```suggestion
        my $data = Mojo::File->new($gzip)->slurp;
```

Reply:
Adapted, this worked nice too. thanks for the suggestion